app/src/test/java/com/saltedge/authenticator/features/authorizations/details/AuthorizationDetailsPresenterTest.kt

```diff
@@ -157,251 +117,55 @@ class AuthorizationDetailsPresenterTest {
     @Throws(Exception::class)
     fun titleTest() {
         val presenter = createPresenter(viewContract = mockView)
-        presenter.setInitialData(
-            connectionId = "1", authorizationId = "",
-            viewModel = null, quickConfirmMode = false
-        )
+        presenter.currentViewModel = null
 
         assertThat(presenter.title, equalTo(TestAppTools.getString(R.string.authorizations_fetching)))
 
-        val authorizationData = createAuthorizationData(1)
-        presenter.setInitialData(
-            connectionId = "1",
-            authorizationId = "",
-            viewModel = authorizationData.toAuthorizationViewModel(connection1),
-            quickConfirmMode = false
-        )
+        val authorization = viewModel1
+        presenter.currentViewModel = authorization
 
         assertThat(presenter.title, equalTo("title1"))
-
-        presenter.onViewClick(R.id.negativeActionView)
-
-        assertThat(
-            presenter.title,
-            equalTo(TestAppTools.getString(R.string.authorizations_processing))
-        )
     }
 
     @Test
     @Throws(Exception::class)
     fun descriptionTest() {
         val presenter = createPresenter(viewContract = mockView)
-        presenter.setInitialData(
-            connectionId = "1", authorizationId = "",
-            viewModel = null, quickConfirmMode = false
-        )
+        presenter.currentViewModel = null
```

Comment:
Can we use this in the method with annotation `@Before`?

Reply:
no